### PR TITLE
PFM-987 :: Configured Jenkins Builds for UI

### PR DESCRIPTION
**Pull request by jagankumar-egov**

### CODE REVIEW

File: build/build-config.yml [hunk 1]
```diff
@@ -26,3 +26,8 @@ config:
         image-name: "estimate-service"
 
 # frontend
+  - name: builds/digit-works/frontend/micro-ui/works
+    build:
+      - work-dir: frontend/micro-ui/
```
Comment (rushang7-eGov) [inline build/build-config.yml:31]: Why does work-dir does not contain /works but name does contain micro-ui/works? 
Is it by mistake?

File: build/build-config.yml [hunk 2]
```diff
@@ -26,3 +26,8 @@ config:
...
+        dockerfile: frontend/micro-ui/web/docker/Dockerfile
+        image-name: works
```
Comment (rushang7-eGov) [inline build/build-config.yml:33]: Don't give a generic image name like works. Add some suffix so that the frontend project is clearly identifiable.

**Review by rushang7-eGov — CHANGES_REQUESTED** [review]

File: build/build-config.yml (hunk 1 above)
Comment (jagankumar-egov) [reply build/build-config.yml:31]: in previous repo Digit-dev also we have given. the build image 
name over there

**Review by jyotishk-eGov — APPROVED** [review]

File: build/build-config.yml (hunk 2 above)
Comment (jagankumar-egov) [reply build/build-config.yml]: added

**Review by jyotishk-eGov — APPROVED** [review]

**Review by rushang7-eGov — APPROVED** [review]

**Pull request merged**